components/atoms/button/ImageUploadButton.tsx

```
@@ -0,0 +1,29 @@
+interface Props {
+  onButtonClick: () => void;
+  onInputChange: (e: ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const ImageUploadButton = forwardRef<HTMLInputElement, Props>(
+  ({ onButtonClick, onInputChange }, ref) => {
+    return (
+      <div
+        className="flex-center size-40 border border-dashed cursor-pointer"
+        onClick={onButtonClick}
+      >
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)